### ISSUE TITLE
Add Trap Rate Correction

### ISSUE THREAD

**Issue by ryankinzer** [KevinSee/damEscapement#22]
We need to adjust the trap valid ruleset. Currently it evaluates if the actual trap catch is within the expected catch using the window count and the estimated trap rate.  However, we have not adjusted the window count for night time passage and during periods of high night-time passage we are labelling the trap as invalid.

We should also consider changing the rule set to be based on sample sizes and not the window to keep the two streams of information independent. For example, the trap is invalid if the number of possible tags is less than 30 and/or the trapped tags is less than 7.

**Comment by KevinSee** [comment]
Makes sense to me that we should adjust the window count for night passage before checking about invalid trap data. 

I hear you about trying to keep the two streams of information independent. Just not sure how to best keep the trap info completely separate from the window counts, since for a given trap rate you'd expect more fish in the trap during periods when window counts are high, and I'm not sure how else to identify high/low periods of overall passage. But very open to the discussion.
